src/Nethermind/Nethermind.Facade/Filters/LogOperators.cs

```diff
@@ -0,0 +1,93 @@
+using System.Collections.Generic;
+using Nethermind.Core.Collections;
+
+namespace Nethermind.Blockchain.Filters.Topics
+{
+
+    public class LogOperators<Element>
+    {
+
+        public static IEnumerable<Element> Intersect(IEnumerator<Element>[] enumerators)
```

Comment:
Methods in this class would benefit from documentation as they are a bit confusing what they want to achieve.